.travis.yml

```diff
@@ -99,7 +99,7 @@ script:
   - echo $PATH
   - echo $PYTHONPATH
   - pip install . --prefix=$INSTALL_DIR
-  - python setup.py test
+  - python setup.py pytest
```

Comment:
Is this really the way to do it? Could we not configure it so that `python setup.py test` still runs the test suite? That is at least how we have done it in our other projects...

Reply:
Sure, seems like a good idea, I've updated accordingly by adding setup.cfg, do you agree with the changes?